src/OrbitQt/main.cpp

```diff
@@ -66,6 +67,8 @@ ABSL_DECLARE_FLAG(bool, local);
 ABSL_DECLARE_FLAG(bool, devmode);
 ABSL_DECLARE_FLAG(bool, nodeploy);
 
+ABSL_FLAG(bool, clear_source_paths_mappings, false, "Clear all the stored sources paths mappings");
```

Comment:
```suggestion
ABSL_FLAG(bool, clear_source_paths_mappings, false, "Clear all the stored source paths mappings");
```

(Which is also what you use in other places.)

Reply:
Right. Thanks.